lib/plain/plain-client.ts

```diff
@@ -23,6 +26,7 @@ export const createPlainClient = (params: ClientParams, defaults?: DefaultParams
     },
     user: {
       getManyForSpace: wrap(wrapParams, endpoints.user.getManyForSpace),
+      getCurrent: wrapHttp(http, endpoints.user.getCurrent),
```

Comment:
I think we can still use `wrap` and it's up to a method use or not to use params?

Reply:
There would be a mandatory `params` popping up in the typings.

Reply:
got it